about/about.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+    <title> Code Comp </title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="shortcut icon" type="image/png" href="favicon.png"/>
+
+  </head>
+
+    <!--First section-->
+
+    <div style="background-image: url(./images/backpic.png);
+      background-size: cover; height:500px; padding-top:80px;
+      text-align:left;">
+    <h1 style= >About Us</h1>
+    
+    </div>
+
+    <!--Second section-->
+    <img src="picture.jpg" style="height:600px; margin:100px;
+      float: left;" alt=" A picture related to teamwork." />
+    <div style="height:600px; margin:100px;">
+      <h2>Helping coders to gain problem solving and coding skills through coding competition. </h2>
+      <br><br>
+      <p style="line-height: 2.0; font-size:20px;">
+        As First-year Software Engineering students committed to learning new skills and knowledge, our mission 
+        in this project is to make the users gain critical thinking by being creative and focused on their goals. 
+        By solving and creating challenges, the users will discover ways of looking at problems from different 
+        aspects by finding solutions, thinking systematically and productively, and finding the most 
+        reasonable solution in short, logical reasoning.
+        
+        </p>
+      <p style="line-height: 2.0; font-size:20px;">Take a look at our GitHub account <a
+          href="https://github.com/code-comp/code-comp.github.io">GitHub </a></p>
+    </div>
+    <!-- Third Section-->
```

Comment:
Since this is a section, it should use a `<section>` tag instead of a `<div>` since that is more semantic